target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -28,6 +28,12 @@ buffalo,whr-g301n)
 	ucidef_set_led_switch "lan3" "LAN3" "$boardname:green:lan3" "switch0" "0x08"
 	ucidef_set_led_switch "lan4" "LAN4" "$boardname:green:lan4" "switch0" "0x10"
 	;;
+engenius,ews511ap)
+	ucidef_set_led_netdev "lan1" "LAN1" "$boardname:blue:lan1" "eth0"
+	ucidef_set_led_netdev "lan2" "LAN2" "$boardname:blue:lan2" "eth1"
+	ucidef_set_led_wlan "wlan2g" "WiFi 2.4GHz" "$boardname:green:wlan2g" "phy1tpt"
```

Comment:
please drop the wireless leds here as well. The triggers are already set via the dts file.

Reply:
Done. Thanks for your reminder.